src/components/ComponentParts/RedButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  link: string;
+  buttonMessage: string;
+  buttonType: 'submit' | 'reset' | 'button' | 'menu';
+  onClick?: () => void;
+}
+
+export const RedButton = ({ link, buttonType, buttonMessage, onClick }: Props) =>
```

Comment:
Nazwa `RedButton` nie do końca moim zdaniem jest poprawna. A co jeśli przyjdzie Klient i stwierdzi, że on chce mieć jednak te przyciski niebieskie? Powinniśmy zmienić tylko w `theme` wersję koloru, a nie zmieniać nazwę komponentu. A tak, to zostanie komponent o nazwie `RedButton` który będzie niebieski :)

Reply:
Zmienione na DefaultSubmitButton.
DefaultButton wydaje mi się zbyt mało dokładne.